std/math.d

```diff
@@ -8131,10 +8261,10 @@ if (isIntegral!(F) && isIntegral!(G))
 }
 
 /**Computes integer to floating point powers.*/
-real pow(I, F)(I x, F y) @nogc @trusted pure nothrow
+Unqual!F pow(I, F)(I x, F y) @nogc @trusted pure nothrow
 if (isIntegral!I && isFloatingPoint!F)
 {
-    return pow(cast(real) x, cast(Unqual!F) y);
+    return pow(cast(Unqual!F) x, cast(Unqual!F) y);
```

Comment:
I would suggest changing the signature to `F pow(I, F)(const I, const F)` and also dispensing with the `Unqual!F` in the body.